packages/studio-base/src/components/AppBar/AppMenu.tsx

```
@@ -89,9 +94,9 @@ export function AppMenu(props: AppMenuProps): JSX.Element {
         label: t("openLocalFile"),
         key: "open-file",
         onClick: () => {
-          dialogActions.dataSource.open("file");
           handleAnalytics("open-file");
           handleNestedMenuClose();
+          openLocalFile().catch(console.error);
```

Do we want to surface these errors to the user? Seems like this is a good way to end up with an error you cannot see.

---

Perhaps although we've been doing the same thing in DataSourceDialog for quite some time.